aerofiles/seeyou/reader.py

```diff
@@ -193,3 +221,101 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_task_options(self, fields):
+        if not fields[0] == "Options":
+            return
+
+        task_options = {
+            'no_start': None,
+            'task_time': None,
+            'wp_dis': False,
+            'near_dis': None,
+            'near_alt': None,
+            'min_dis': False,
+            'random_order': False,
+            'max_pts': None,
+            'before_pts': None,
+            'after_pts': None,
+            'bonus': None
+        }
+
+        for field in fields[1:]:
+            field_type = field.split("=")[0]
+            field_entry = field.split("=")[1]
+
+            if field_type == 'NoStart':
+                task_options['no_start'] = field_entry
+            elif field_type == 'TaskTime':
+                task_options['task_time'] = field_entry
+            elif field_type == 'WpDis':
+                task_options['wp_dis'] = field_entry == "True"
+            elif field_type == 'NearDis':
+                task_options[field_type] = float(field_entry[0:-2])   # take of km
+            elif field_type == 'NearAlt':
+                task_options[field_type] = float(field_entry[0:-1])   # take of m
```

Comment:
that seems to be a bug in the regex. it should match on a period character, not *any* character. essentially the `.` needs to be escaped using `\.`.